tools/gulp/constants.ts

```diff
@@ -29,6 +29,12 @@ export const HTML_MINIFIER_OPTIONS = {
   removeAttributeQuotes: false
 };
 
+/** Packages that will be published to NPM by the release task. */
+export const RELEASE_PACKAGES = [
+  'cdk',
+  'material',
+];
```

Comment:
I don't really like this; I don't see why have a `constants.ts` file full of unrelated things is better than having the constant for release packages defined where it's used.